README.md

```diff
@@ -43,180 +36,32 @@ import "github.com/googollee/go-socket.io"
 
 and use `socketio` as the package name inside the code.
 
-## Breaking changes
```

Comment:
Shouldn't we keep this section? And add the change about .NewServer not returning `error`

Reply:
I suggest to add this information at release note - [here](https://github.com/googollee/go-socket.io/releases) or we can made new file by for all changes. Often, them named is CHANGELOG.md